lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2010 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+/**
+ * @type {Object<string, Object<string, string>>}
+ */
+const stackPacks = {
+  WordPress: {
+    'unused-css-rules': 'Consider reducing, or switching, the number of [WordPress plugins](https://wordpress.org/plugins/) loading unused CSS in your page. To identify plugins that are adding extraneous CSS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the stylesheet. Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage. A plugin should only enqueue a stylesheet if it is actually used on the page.',
+    'uses-webp-images': 'Consider using a plugin/service that will automatically convert your uploaded images to the optimal formats.',
+  },
+}; //require('@lighthouse/stack-packs');
+
+/**
+ * @param {LH.Artifacts} artifacts
+ * @return {Array<LH.StackPacks>}
+ */
+function getStackPacks(artifacts) {
+  /** @type {Array<LH.StackPacks>} */
+  const packs = [{
+    id: 'WordPress',
```

Comment:
I didn't expect that the icon/detectedLibs/id would be defined here in core.. I had expected those would be pulled in along with the advice strings. 

Is there a reason you went that direction? I'm open to it.. just curious.

Reply:
no real reason, I copied it from the issue that's all but it's indeed probably better to have this inside stack packs;
```
stackPacks: {
   'WordPress': {
      id: 'WordPress',
      icon: 'data:image/...'
      advice: {
        'unused-css-rules': 'Consider reducing, or switching, the number of [WordPress plugins](https://wordpress.org/plugins/) loading unused CSS in your page. To identify plugins that are adding extraneous CSS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the stylesheet. Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage. A plugin should only enqueue a stylesheet if it is actually used on the page.',
        'uses-webp-images': 'Consider using a plugin/service that will automatically convert your uploaded images to the optimal formats.',
      }
  }
}
```